### ISSUE TITLE
Unable to use the submit script

### ISSUE THREAD

**Issue by yvgny** [LPD-EPFL/CS453-2019-project#6]
Hello,

When I try to submit my files to the server, I get the following output trace : 
```
Your submission has been queued on the server. Please wait.
Your submission is compiling on the server. Please wait.
g++ -Wall -Wextra -Wfatal-errors -O2 -std=c++17 -fPIC -I../include -c -o bloom_filter.cpp.o bloom_filter.cpp
g++ -Wall -Wextra -Wfatal-errors -O2 -std=c++17 -fPIC -I../include -c -o tm.cpp.o tm.cpp
g++ -shared -o ../260391.so  ./bloom_filter.cpp.o ./tm.cpp.o
Makefile:46: recipe for target '../260391.so' failed
/usr/local/bin/ld: cannot open output file ../260391.so: Permission denied
collect2: error: ld returned 1 exit status
make: *** [../260391.so] Error 1
Your submission could not compile on the server.
```

However, it compiles locally (tested on macOS and Ubuntu). I also applied the last patches available on this repo (until commit 6a07a5dcd67966008c4e5965f149832e8e5e2e16).

Am I doing something wrong ?

Thank you very much !

**Comment by srlt** [comment]
TL;DR: Zip the **whole directory** containing your code, not directly the Makefile and sources.

The compilation and execution are both sandboxed, and `make` cannot write inside `../`.
When you zip the whole directory containing your code (as indicated in the README), then the library gets output at `your_directory/../260391.so` = `./260391.so`, where `make` is allowed to write.

I let you close this issue if this solved your problem.

**Comment by yvgny** [comment]
Indeed, it solved the issue, thanks a lot !
